documentation/asciidoc/accessories/camera/raspicam.adoc

```diff
@@ -1332,3 +1360,44 @@ The applications described here will return a standard error code to the shell o
 | Application terminated by Ctrl-C
 |===
 
+=== Long Exposures
+
+The maximum exposure times of the three official Raspberry Pi cameras can be found in xref:camera.adoc#maximum-exposure-times[this table].
+
+Due to the way the ISP works, by default asking for a long exposure can result in the capture process taking up to 7 times the exposure time, so a 200 second exposure on the HQ camera could take 1400 seconds to actually return an image. This is due to the way the camera system works out the correct exposures and gains to use in the image, using it's AGC (automatic gain control) and AWB (automatic white balance) algorithms. The system needs a few frames to calculate these numbers in order to produce a decent image. When combined with frame discards at the start of processing (in case they are corrupt), and the switching between preview and captures modes, this can result in up to 7 frames needed to produce a final image. With long exposures, that can take a long time.
```

Comment:
This paragraph is duplicated in longexp.adoc ?

Reply:
Ah, I think that (longexp.adoc) is another file where the contents has moved (to raspicam.adoc) so longexp.adoc needs to be deleted.

Reply:
Looks like it's not included anymore. I'll delete it.

<img width="664" alt="Screenshot 2021-09-23 at 17 16 14" src="https://user-images.githubusercontent.com/53625/134545140-6a76f927-6ea1-4d9d-98db-243c06d16f10.png">

Reply:
Deleted in 6c41b93.